driver-core/src/main/java/com/datastax/driver/core/policies/ErrorAwarePolicy.java

```diff
@@ -201,12 +199,13 @@ public Builder withInclusionThreshold(double errorsPerSecThreshold) {
          * <p/>
          * The default implementation will exclude from the error counting, the following exception types :
          * <ul>
-         *     <li>{@link ReadTimeoutException}</li>
-         *     <li>{@link WriteTimeoutException}</li>
-         *     <li>{@link UnavailableException}</li>
-         *     <li>{@link ReadFailureException}</li>
-         *     <li>{@link WriteFailureException}</li>
+         * <li>{@link ReadTimeoutException}</li>
```

Comment:
hrmm, i like that behavior much better too (keeping the indention).  I'll look into my autoformat settings.